src/sha1/sha1.c

```diff
@@ -267,10 +267,10 @@ main(int argc, char **argv)
         SHA1Update(&ctx, buf, BUFSIZE);
     SHA1Final(hash, &ctx);
 
-    Platform::LogMessage("SHA1=");
```

Comment:
better not poke in external dependencies. Those are tests anyway, they are never run.

Reply:
Yep, which is why I changed it back to printf.